habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -391,9 +398,7 @@ def _eval_checkpoint(
         self.metric_uuid = measure_type(None, None)._get_uuid()
 
         observations = self.envs.reset()
-        batch = batch_obs(observations)
-        for sensor in batch:
```

Comment:
@erikwijmans was is a duplication with `batch_obs` functionality?

Reply:
I added the ability to have batch obs move things to a given device as that cleaned things up